src/models/session/logic.ts

```diff
@@ -94,6 +96,56 @@ export function sortBreadcrumbs(
   return orderedBreadcrumbs;
 }
 
+export function flattenBreadcrumbs(
+  orderedBreadcrumbs: OrderedBreadcrumbs,
+): Breadcrumbs {
+  return orderedBreadcrumbs!.reduce((acc, crumb) => {
+    acc[crumb.id] = {
+      auto: crumb.autoAnswered,
+      answers: crumb.answers,
+      data: crumb.data,
+      override: crumb.override,
+      feedback: crumb.feedback,
+    };
+    return acc;
+  }, {} as Breadcrumbs);
+}
+
+export function computePassport(breadcrumbs: OrderedBreadcrumbs): IPassport {
```

Comment:
TODO: Test this using existing `computePassport()` tests in - 
 - `planx-new/editor.planx.uk/src/pages/FlowEditor/lib/__tests__/preview.test.ts`
 - `planx-new/editor.planx.uk/src/pages/FlowEditor/lib/__tests__/overrideAnswer.test.ts`
 - `planx-new/editor.planx.uk/src/pages/FlowEditor/lib/__tests__/granularity.test.ts`